src/wp-content/themes/twentyfifteen/functions.php

```diff
@@ -250,6 +250,73 @@ function twentyfifteen_setup() {
 			)
 		);
 
+		// Add support for custom color scheme.
```

Comment:
Was this file accidentally added to the PR?

Reply:
fixed thanks